src/Imgix/UrlBuilder.php

```diff
@@ -27,31 +25,27 @@ class UrlBuilder {
     const MAX_WIDTH = 8192;
     const SRCSET_WIDTH_TOLERANCE = 0.08;
 
-    // constants cannot be dynamically assigned; keeping as a class variable instead
-    private $targetWidths;
-
     public function __construct($domain, $useHttps = true, $signKey = "", $includeLibraryParam = true) {
 
         if (!is_string($domain)) {
             throw new \InvalidArgumentException("UrlBuilder must be passed a string domain");
         }
 
         $this->domain = $domain;
-        $this->validateDomain($this->domain);        
+        $this->validateDomain($this->domain);
 
         $this->useHttps = $useHttps;
         $this->signKey = $signKey;
         $this->includeLibraryParam = $includeLibraryParam;
-        $this->targetWidths = $this->targetWidths();
```

Comment:
Removing this is related to the class property being removed above, just for clarification.